### PR TITLE
Lusine/web front end/16 jun2021

### PR DESCRIPTION
**Pull request by LusineShahbazyan**
Make script for registration

### CODE REVIEW

File: Lusine_Shahbazyan/web/registration/script.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+let users = [];
+
+let user1 = new Admin("uname1", "password1", 30, "Mary", "Makaryan");
+if(registration(users, user1))
+{
+    users.push(user1);
+}
+let user2 = new Guests("uname2", "password2", 12, "Ani", "Andreasyan");
+if(registration(users, user2))
```
Comment (andreasyan-liana) [inline Lusine_Shahbazyan/web/registration/script.js]: registration function should register users (Not just validate).

File: Lusine_Shahbazyan/web/registration/script.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        {
+            console.log("There is user with this password and username");
+            return false;
+        }
+    }
+    console.log("Registration completed successfully")
+    return true;
+}
+
+function authentication(array)
```
Comment (andreasyan-liana) [inline Lusine_Shahbazyan/web/registration/script.js]: Please try to update authenticate function to have the following form
`authenticate(username, password): return logged in user data`

Comment (LusineShahbazyan) [reply Lusine_Shahbazyan/web/registration/script.js]: I have update authenticate function

File: Lusine_Shahbazyan/web/registration/script.js (hunk 1 above)
Comment (LusineShahbazyan) [reply Lusine_Shahbazyan/web/registration/script.js]: I have update registration function

**Pull request merged**